supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -323,12 +323,13 @@ There is no functional difference between the first server that was installed an
 *See also*:
 
 [[ingress]]
-==== image:images/yes.png[yes] Ingress (noun)
-*Description*: In Red Hat OpenShift, _Ingress_  is an API object developers can use to expose services through an HTTP(S) aware load balancing and proxy layer through a public DNS entry. The Ingress resource might further specify TLS options and a certificate, or specify a public CNAME that the OpenShift Ingress Controller should also accept for HTTP and HTTPS traffic. An administrator typically configures their Ingress Controller to be visible outside the cluster firewall and might also add additional security, caching, or traffic controls on the service content.
+==== image:images/yes.png[yes] ingress (noun, adjective)
+*Description*: In Red Hat OpenShift, _Ingress_  is a Kubernetes API object that developers can use to expose services within the cluster through an HTTP(S) load balancer and a proxy layer using a public DNS entry. The `Ingress` resource defines the cluster-wide configuration for ingress traffic, and provides the ability to specify TLS options, a certificate, or a public CNAME that the OpenShift `IngressController` object can accept for HTTP(S) traffic. Additionally, _ingress_ can also be used to describe the incoming direction of network traffic. In Red Hat OpenShift, for example, this traffic is described as entering (ingress) or leaving (egress) an OpenShift cluster.
```

Comment:
Sorry, I just notice a small issue. Could you please add a word to this sentence?

> In Red Hat OpenShift, _Ingress_ is a Kubernetes API object that developers can use to expose services within the cluster through an HTTP(S) load balancer and a proxy layer using a public DNS entry.

Please add "by" before "using a public DNS entry."

Reply:
Addressed :)

Reply:
The timing is excellent as I just stumbled across CNAME. It means "Canonical Name". Maybe put this in parentheses? JUst more a case of keeping consistent with spelling out full terms on first use.

Reply:
When I search the OCP docs repo, CNAME is never expanded in parentheses so I don't really know if its necessary to do that here in this definition.

Reply:
@redhat-documentation/ccs-style-council , WDYT? OUt of context or good for consistency of rules?

Reply:
I vote out of context